include/OpenColorIO/OpenColorABI.h.in

```diff
@@ -77,6 +69,14 @@ OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
 		#define OCIOEXPORT
 	#endif
     #define OCIOHIDDEN
+#elif defined __GNUC__
+    #if __GNUC__ >= 4
```

Comment:
Just as an aside (not necessarily for the author of this patch, unless you are particularly inclined to fix it now), this can be simplified a bit because now that we're C++11, there is no need to consider gcc versions < 4.x.